lib/takeaway.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'text'
+require 'time'
+
+class Takeaway
+
+  attr_reader :order, :total
+
+  def initialize(menu = Menu.new, order = Order.new, text = Text.new)
+    @menu = menu.parse_file
+    @order = order
+    @total = 0
+    @text = text
+  end
+
+  def order_food(item_number, quantity)
+    item_number -= 1
+    @order.select(item_number, @menu, quantity)
+    @total += @order.pad[-1][:cost] * quantity
+  end
+
+  def remove_food(item_number, quantity)
+    item_number -= 1
+    @total -= @menu[item_number][:cost] * quantity
+    @order.remove(@menu[item_number][:name], quantity)
+  end
+
+  def display_menu
```

Comment:
Love the separate methods for displaying things to the user. Might be a good idea to have a Display class that deals with communication with the user?